src/nemo_evaluator/environments/gym_protocol.py

```
@@ -106,16 +109,54 @@ def extract_prompt_from_rcp(rcp: dict[str, Any]) -> str:
     return ""
 
 
-def messages_from_rcp(rcp: dict[str, Any]) -> list[dict[str, str]]:
-    """Build a messages list from responses_create_params.input."""
+def _as_text(content: Any) -> str:
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `content`

(ANN401)